lib/push.js

```diff
@@ -177,6 +181,39 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies every device in the account except the current one that the
+     * password was changed
```

Comment:
Why "except the current one"?  I assume it's because the current one already knows about the password-reset, thanks to a webchannel message from content-server.  But counting on that seems vaguely fragile to me.  Is there a way for the device to sanely handle being notified twice, by both push message and webchannel message?

(FWIW I'm happy for the answer to be "no, not really" and leave this as-is - just want to make sure we consider it)